python/lsst/pipe/tasks/background.py

```diff
@@ -773,11 +825,310 @@ def getStatsImage(self):
+    translation from warps to tract and back only requires geometric
+    transformations in the warped pixel plane.
+    """
+
+    xBin = Field(dtype=float, default=500, doc="Bin size in x")
+    yBin = Field(dtype=float, default=500, doc="Bin size in y")
+    minFrac = Field(dtype=float, default=0.1, doc="Minimum fraction of bin size for good measurement")
+    mask = ListField(
+        dtype=str,
+        doc="Mask planes to treat as bad",
+        default=["BAD", "SAT", "INTRP", "DETECTED", "DETECTED_NEGATIVE", "EDGE", "NO_DATA"],
+    )
+    interpolation = ChoiceField(
+        doc="how to interpolate the background values. This maps to an enum; see afw::math::Background",
+        dtype=str,
+        default="AKIMA_SPLINE",
+        optional=True,
+        allowed={
+            "CONSTANT": "Use a single constant value",
+            "LINEAR": "Use linear interpolation",
+            "NATURAL_SPLINE": "cubic spline with zero second derivative at endpoints",
+            "AKIMA_SPLINE": "higher-level nonlinear spline that is more robust to outliers",
+            "NONE": "No background estimation is to be attempted",
+        },
+    )
+    doSmooth = Field(dtype=bool, default=False, doc="Do smoothing?")
+    smoothScale = Field(dtype=float, default=2.0, doc="Smoothing scale, as a multiple of the bin size")
+    binning = Field(dtype=int, default=64, doc="Binning to use for warp background model (pixels)")
+
+
+class TractBackground:
+    """
+    As FocalPlaneBackground, but works in warped tract coordinates
+    """
+    @classmethod
+    def fromSimilar(cls, other):
+        """Construct from an object that has the same interface.
+
+        Parameters
+        ----------
+        other : `TractBackground`-like
+            An object that matches the interface of `TractBackground`
+            but which may be different.
+
+        Returns
+        -------
+        background : `TractBackground`
+            Something guaranteed to be a `TractBackground`.
+        """
+        return cls(other.config, other.tract, other.dims, other.transform, other._values, other._numbers)
+
+    def __init__(self, config, skymap, tract, values=None, numbers=None):
+        """Constructor
+
+        Developers should note that changes to the signature of this method
+        require coordinated changes to the `__reduce__` and `clone` methods.
+
+        Parameters
+        ----------
+        config : `TractBackgroundConfig`
+            Configuration for measuring tract backgrounds.
+        skymap : `lsst.skymap.ringsSkyMap.RingsSkyMap`
+            Skymap object
+        tract : `int`
+            Placeholder Tract ID
+        transform : `lsst.afw.geom.TransformPoint2ToPoint2`
+            Transformation from tract coordinates to warp coordinates.
+        values : `lsst.afw.image.ImageF`
+            Measured background values.
+        numbers : `lsst.afw.image.ImageF`
+            Number of pixels in each background measurement.
```

Comment:
`__init__` parameter docs should go in the class docstring.  I don't think a docstring on an `__init__` actually goes anywhere useful.  Or at least it doesn't end up in all of the useful places.